modules/ztp-install-openshift-pipelines.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="install-the-openshift-pipelines-operator_{context}"]
+= Installing the {product-title} Pipelines Operator
+
+Follow this guidance to install the OpenShift Pipelines Operator that is used to run the pipeline.
+
+.Prerequisites
+
+* An {product-title} hub cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install `git`. For guidance on installing `git`, see link:https://github.com/git-guides/install-git[Install Git].
+
+.Procedure
+
+. Export the `KUBECONFIG` environment variable.
++
+[source,terminal]
+----
+$ export KUBECONFIG=<path_to_kubeconfig>/kubeconfig
+----
+
+. Run the following bash script `bootstrap.sh` with the `KUBECONFIG` as a parameter to install the OpenShift Pipelines Operator.
++
+[source,terminal]
+----
+$ curl -sLK https://raw.githubusercontent.com/rh-ecosystem-edge/ztp-pipeline-relocatable/main/pipelines/bootstrap.sh | bash -s -- ${KUBECONFIG}
+----
++
+This script:
+
+* Installs the `tkn` CLI. This tool manages {product-title} pipelines from a terminal.
+* Clones the link:https://github.com/rh-ecosystem-edge/ztp-pipeline-relocatable[ztp-pipeline-relocatable] pipeline repository.
```

Comment:
We usually need to get our GitHub links checked by @sferich888 - just checking if you've done that?

Reply:
@sferich888 This GitHub repo is an integral part of being able to run the ZTP pipelines and my understanding is it is in an official repo.

Reply:
> What about it makes it 'integral'? Why can't the contents of the repo be reflected in our docs?

We could, indeed! It's a lot of content, tho. The solution and steps are supported, whereas the scripts aren't. They are meant to be read as examples and we need to make it clear that they are community-supported